### PR TITLE
TK224 - Fixed batch name issue

### PR DESCRIPTION
**Pull request by karuna835**

### CODE REVIEW

File: app/apis/btcapi/students.php [hunk 1]
```diff
@@ -332,8 +330,18 @@
...
+						FROM subs
+						INNER JOIN subs_meta ON subs_meta.subs_id=subs.subs_id
+						LEFT JOIN bootcamp_batches ON bootcamp_batches.id=subs_meta.batch_id
+						WHERE subs.user_id = '".$row['id']."' AND
+						subs_meta.bundle_id = '".$row['course_id']."';"
+						);
+		
 			//TK224 - batch name explode using comma seperator and get unique batch name
-			$batches = explode("+", $row["batch_name"]);
+			$batches = explode("+", $get_subs_id[0]['batch_name']);
```
Comment (kailasb) [inline app/apis/btcapi/students.php:344]: add check for if batch is not found any of the user

File: app/apis/btcapi/students.php [hunk 2]
```diff
@@ -332,8 +330,18 @@
...
 			$batch_array = array();
 			foreach ($batches as $batch) {
               $result = json_decode($batch, true);
```
Comment (kailasb) [inline app/apis/btcapi/students.php:347]: add check for if batch record is empty

**Review by kailasb — APPROVED** [review]
code change is proper

File: app/apis/btcapi/students.php (hunk 2 above)
Comment (karuna835) [reply app/apis/btcapi/students.php]: Added checks for batch record empty or not

**Pull request merged**